src/scenekit.cs

```diff
@@ -2903,14 +3340,17 @@ interface SCNView : SCNSceneRenderer, SCNTechniqueSupport {
 		SCNAntialiasingMode AntialiasingMode { get; set; }
 
 #if XAMCORE_2_0
-		[Watch (4, 0), TV (11, 0), Mac (10, 13, onlyOn64: true), iOS (11, 0)]
+		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
```

Comment:
Why was onlyOn64 removed?

Reply:
@timrisi it breaks the build for classic. I'm not sure why/when those were added in master and why I asked your review ;-)